feature/default_sets.go

```diff
@@ -100,7 +100,7 @@ var defaultSetDesc = setDesc{
 		SetInit:    {}, // I
 		SetNodeAnn: {}, // N
 	},
-	lnwire.ExperimentalEndorsementOptional: {
+	lnwire.ExperimentalAccountabilityOptional: {
```

Comment:
nit: [one mention](https://github.com/lightningnetwork/lnd/blob/master/routing/route/route.go#L529) of endorsement left in comment.